.eslintrc.json

```diff
@@ -83,6 +83,77 @@
             "rules": {
                 "no-console": ["off"]
             }
+        },
+        {
+            "files": ["package.json"],
+            "parser": "jsonc-eslint-parser",
+            "plugins": [],
+            "extends": ["plugin:jsonc/recommended-with-jsonc"],
+            "rules": {
+                "notice/notice": "off",
+                "jsonc/sort-keys": [
+                    "warn",
+                    {
+                        "pathPattern": "^$",
+                        "order": [
+                            "$schema",
+                            "private",
+                            "publishConfig",
```

Comment:
Maybe nit-picky, or I'm just used to it, but I do like to see `name` and `version` above `publishConfig`. What do you think? 😛

Reply:
I'm flexible - this was based on the API docs I think. It's been a while.

Reply:
Update: I ended up removing this as out of scope because it wasn't as straight forward as I'd hoped.